File: go/uri-online-judge/1011.go
package main

import (
	"fmt"
	"math"
)

var radius float64

func main() {
	fmt.Scanf("%f", &radius)
	fmt.Printf("VOLUME = %.3f\n", (4.0/3)*3.14159*math.Pow(radius, 3))
}

File: javascript/uri-online-judge/1011.js
const input = require('fs').readFileSync('/dev/stdin', 'utf-8');
const number = input.split('\n');

const sphereVolume = (radius) => {
  return (4/3) * 3.14159 * Math.pow(radius, 3);
}

const radius = number.shift();
const volume = sphereVolume( radius );

console.log(`VOLUME = ${ volume.toFixed(3) }`);
